packages/mdc-toolbar/README.md

```diff
@@ -20,13 +19,32 @@ Wrap the items with `mdc-toolbar` class in following way:
 
 ```html
 <header class="mdc-toolbar">
-  <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
-    <a class="material-icons">menu</a>
-    <span class="mdc-toolbar__title">Title</span>
-  </section>
+  <div class="mdc-toolbar__row">
+    <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
+      <a class="material-icons">menu</a>
+      <span class="mdc-toolbar__title">Title</span>
+    </section>
+  </div>
 </header>
 ```
 
+You can wrap second row such as mdc-tabs within mdc-toolbar:
```

Comment:
Perhaps something like:

MDC Toolbars can accommodate multiple rows using the wrapper `mdc-toolbar__row`